resume_code/override.css

```
@@ -0,0 +1,443 @@
+    margin: var(--viewer-pages-spacing) 0;
+  }
+}
+ @media screen and (min-width: 8.5in) {
+  /* not a mobile */
+  body {
+    margin: auto;
+    width: unset;
+  }
+}
+
+.pagedjs_page:not(:first) {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":first" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)